comp/snmpscan/impl/devicescan.go

```diff
@@ -54,3 +57,72 @@ func gatherPDUs(snmp *gosnmp.GoSNMP) ([]*gosnmp.SnmpPDU, error) {
 	}
 	return pdus, nil
 }
+
+func (s snmpScannerImpl) ScanDeviceAndSendData(connParams *snmpparse.SNMPConfig, logger log.Component, namespace string) error {
+	// Establish connection
+	snmp, err := snmpparse.NewSNMP(connParams, logger)
+	if err != nil {
+		return err
+	}
+	deviceID := namespace + ":" + connParams.IPAddress
+	// Since the snmp connection can take a while, start by sending an in progress status for the start of the scan
+	// before connecting to the agent
+	fmt.Printf("Initializing scan for device: %s\n", deviceID)
```

Comment:
We should not use print line within the component, otherwise this will mess up the log line written by the agent.
Could we use the `s.log` here ?

Reply:
And if it's just for the cli, the print should be in the cli code